overrides/components/page-footer/component.tsx

```diff
@@ -195,6 +192,64 @@ export default function PageFooter(props) {
 
   const showModal = modalRevealed && modalsDismissed[currentPage] === false;
 
+  const menuItems = [
+    {
+      path: DATASETS_PATH,
+      text: 'Data Catalog',
+    },
+    {
+      path: ANALYSIS_PATH,
+      text: 'Data Analysis',
+    },
+    {
+      path: STORIES_PATH,
+      text: getString("stories").other,
+    },
+    ...(!!process.env.HUB_URL && !!process.env.HUB_NAME ? [{
+      path: process.env.HUB_URL,
+      text: process.env.HUB_NAME,
+    }]: []),
+    {
+      path: "/visit",
+      text: "Visit the center",
+    },
+    {
+      path: "/teach",
+      text: "Teach",
+    },
+    {
+      path: ABOUT_PATH,
+      text: "About",
+    },
+    {
+      path: "https://docs.google.com/forms/d/1mDgFqUsNv90Js7pERNDbmyN5RksztIy5ZZojWD0n5Pg",
+      text: "Contact Us",
+    },
+  ]
+
+  const createMenu = () => (
+    <FooterMenu>
+      {menuItems.map((item, i) => {
+        if ((item.path == process.env.HUB_URL && item.text == process.env.HUB_NAME) || item.text == "Contact Us") {
```

Comment:
I'm not too worried about scalability in this case because the footer menu shouldn't be growing exponentially in size 😄  but what you shared does look cleaner and simplifies! Will do that! I think this map is a bit of an improvement from what existed before